metabase/metabase
# Fix wrong icon colors

From a [Slack discussion](https://metaboat.slack.com/archives/C063Q3F1HPF/p1709097608250029).

### Description

There are 2 wrong icon colors:
1. The official collection icon in collection view.
2. The verified model icon.

### How to verify

1. Create a collection inside "Our analytics" and make it official.
2. Create a model and verify it.
3. Go to admin settings > appearance settings > Change the main color (first color) and refresh the page
4. Go to Our analytics collection view.
5. See that the official collection icon color should be yellow and the verified model icon color should be the main color set in step 4)

### Demo

#### Before
<img width="1452" alt="Screenshot 2024-02-28 at 1 59 55 PM" src="https://github.com/metabase/metabase/assets/1937582/74bd7d5a-418f-498c-945b-797365148b85">
<img width="1452" alt="Screenshot 2024-02-28 at 2 00 05 PM" src="https://github.com/metabase/metabase/assets/1937582/f72ef1d7-b221-46ee-a45b-766d467dab69">


#### After

<img width="1452" alt="Screenshot 2024-02-28 at 1 51 45 PM" src="https://github.com/metabase/metabase/assets/1937582/6447ea65-e913-4ad0-a548-72bdb76972ce">
<img width="1451" alt="Screenshot 2024-02-28 at 1 51 33 PM" src="https://github.com/metabase/metabase/assets/1937582/03028953-638c-4ee7-b11a-23a1589e34c9">

### Checklist

- [ ] ~Tests have been added/updated to cover changes in this PR~ there are only color changes
